contrib/ros/worlds/block_world/world.py

```diff
@@ -9,8 +9,12 @@
 
 from contrib.ros.worlds.gazebo import Gazebo
 from contrib.ros.worlds.world import World
-from garage.config_personal import VICON_TOPICS
 import garage.misc.logger as logger
+try:
+    from garage.config import VICON_TOPICS
+except ImportError:
+    logger.log("If you want to use real robot please set "
```

Comment:
"in config_personal.py"

Reply:
Well, I'm a little confused that are we still using config_personal.py instead of config.py.
Should it be:
```
try:
    from garage.config_personal import VICON_TOPICS
except ImportError:
    logger.log("If you want to use real robot please set "
                    "VICON_TOPICS in config_personal.py, otherwise skip this")
```

Reply:
`config.py`:
```python
from garage.config_personal import *
```

`config.py` is a Python file containing configuration information for garage. Any garage module needing configuration information imports config.py and expects all configs to be there.

Some configuration items are either user-specific or need to be overridden by users. Those items are put in `config_personal.py`, which is automatically copied for each user on first run from `config_personal_template.py`. Anything in `config_personal.py` overwrites the default setting in `config.py`

garage maintainers edit `config.py`/`config_personal_template.py` but users should only ever edit their individual `config_personal.py`.